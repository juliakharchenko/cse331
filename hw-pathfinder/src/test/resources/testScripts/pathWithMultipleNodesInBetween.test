# Find a path with Dijkstra's algorithm between two nodes
# that have nodes connected between them.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph B C 7.0
AddEdge simpleGraph C D 4.0

FindPath simpleGraph A D