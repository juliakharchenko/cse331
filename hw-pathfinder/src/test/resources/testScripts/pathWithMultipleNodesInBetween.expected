# Find a path with Dijkstra's algorithm between two nodes
# that have nodes connected between them.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 7.000 from B to C in simpleGraph
added edge 4.000 from C to D in simpleGraph

path from A to D:
A to B with weight 10.000
B to C with weight 7.000
C to D with weight 4.000
total cost: 21.000